src/browser/commands.js

```diff
@@ -30,6 +30,7 @@ export async function focusElem(elem) {
 
 export async function hoverAt(x, y) {
 	await sendMouse({ type: 'move', position: [x, y] });
+	if (window.d2lTest) window.d2lTest.hovering = true;
```

Comment:
Would having the vdiff/pause stuff just add a listener on the window or body for `mouseover` also work? If so, it might be nice for that code to live over there where it belongs.